site-src/geps/gep-713.md

```diff
@@ -0,0 +1,409 @@
+    //
+    // Support: Core
+    //
+    // +kubebuilder:validation:MinLength=1
+    // +kubebuilder:validation:MaxLength=253
+    Name string `json:"name"`
+
+    // SectionName is the name of a section within the target resource. When 
+    // unspecified, this targets the entire resource. In the following
+    // resources, SectionName is interpreted as the following:
+    // * Gateway: Listener Name
+    // * Route: Rule Name
+    // * Service: Port Name
+    //
+    // Support: Core
+    //
+    // +kubebuilder:validation:MinLength=1
+    // +kubebuilder:validation:MaxLength=253
+    // +optional
+    SectionName string `json:"sectionName,omitempty"`
+
+    // Namespace is the namespace of the referent. When unspecified, the local
+    // namespace is inferred. Even when policy targets a resource in a different
+    // namespace, it may only apply to traffic originating from the same 
+    // namespace as the policy.
+    //
+    // Support: Extended
+    //
+    // +kubebuilder:validation:MinLength=1
+    // +kubebuilder:validation:MaxLength=253
+    // +optional
+    Namespace string `json:"namespace,omitempty"`
+    
+    // ClassName is the name of the class this policy should apply to. When
+    // unspecified, the policy will apply to all classes that support it.
+    //
+    // Support: Core
+    //
+    // +kubebuilder:validation:MinLength=1
+    // +kubebuilder:validation:MaxLength=253
+    // +optional
+    ClassName string `json:"className,omitempty"`
+}
+```
+
+### Status
+A new `Policies` field will be added to status on Gateways and Routes. This will
+be a list of `PolicyTargetReference` structs with the fields instead used to
+refer to the Policy resource that has been applied.
+
+### Hierarchy
+Policy precedence is determined in the same direction a request flows. For
+ingress, that means from Gateway to Backend, and for mesh that means from
+Consumer to Backend. 
+
+| Ingress | Sidecar Consumer |
+|-|-|
+| Gateway Namespace | Consumer Namespace |
+| Gateway | Consumer |
+| Route Namespace | Route Namespace |
+| Route | Route |
+| Backend Namespace | Backend Namespace |
+| Backend | Backend |
+
+Some implementations may wish to provide default or enforced values. For default
+values, the hierarchies above would be used. For example, if a default value was
+specified for both a Route and Backend, the value attached to the backend would
+be given precedence. For enforced values, precedence would be given to the
+policy attached higher in the hierarchy.
+
+Each policy resource would include 2 structs within the spec. One with enforced
+values and the other with default values. In the following example, the policy
+attached to the Gateway requires cdn to be enabled and provides some default
+configuration for that. The policy attached to the Route changes the value for
+one of those fields (includeQueryString).
+
+```yaml
+kind: GKEServicePolicy # Example of implementation specific policy name
+spec:
+  enforce:
```

Comment:
During the 7/12 meeting @youngnick suggested to rename this as `override`, and someone else suggested `overrideChild` or `overrideChildren`.

Reply:
also +1 to `override`, which I prefer to `overrideChild` and `overrideChildren` because of key verbosity.

Reply:
Updated, thanks!